frontends/web/src/routes/account/send/send.tsx

```diff
@@ -270,17 +270,24 @@ class Send extends Component<Props, State> {
       amountError: undefined,
       feeError: undefined,
     });
-    if (this.sendDisabled()) {
+    if (this.sendDisabled() || this.state.feeTarget === undefined) {
```

Comment:
Won't checking `feeTarget === undefined` be redundant here? As it's already evaluated the same way in `sendDisabled`?

Reply:
absolutely, but TypeScript ignores that feeTarget has been already checked in the sendDisabled function and complains, I'll check if I find an elegant solution.

Reply:
Found an elegant solution and typescript found a bug, fixed in separate commit.

PTAL